src/main/java/com/sintern/domain/entity/Application.java

```diff
@@ -32,4 +32,44 @@
 
     String description;
     LocalDateTime dateOfSubmission;
+
+    public UUID getId() {
```

Comment:
Why don't we use Lombok annotations for Getters and Setters?

Reply:
Actually, I didn't see the annotations and I will refactor this